torch/quantization/quantize.py

```diff
@@ -73,7 +73,11 @@ def _observer_forward_hook(self, input, output):
     """
     return self.activation_post_process(output)
 
-def add_observer_(module, non_leaf_module_list=None, device=None):
+def _observer_forward_pre_hook(self, input):
+    self.activation_pre_process(input[0])
```

Comment:
why is this `input[0]` can you add a comment? also maybe you can remove `return None` and add a comment why we don't need to return as well